### PR TITLE
Edit doc landing page and add one additional doc page

### PR DESCRIPTION
**Pull request by Stanislava101**
The new doc file is created to help developers to trace webhooks sent by Github
and receive them without exposing user app to the internet.

The docs/index.md has documentation both for contributers and users
and it is a good idea to have them regroup in to two groups for clarity.

Fix for issue https://github.com/vmware/precaution/issues/243.

Signed-off-by: Stanislava Georgieva stanislava.krasimirovaa@abv.bg

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -19,18 +19,31 @@ GitHub integration is made through the GitHub app interface and the checks API (
...
 
 - [Initial setup](initial_setup.md)
 - [Setting up Precaution as branch protection rule](branch_protection_rule.md)
 - [False positives and how to handle them](false_positivies.md)
+
+## Contributers documentation
+
+If you are enthusiastic about Precaution and want to improve it don't be concerned. Anyone can contribute, whether you’re new to the project or not.
+
+- Bugs: [vmware/precaution/issues](https://github.com/vmware/precaution/issues)
```
Comment (MVrachev) [inline docs/index.md]: Maybe we can move the `Bugs` link where are the Documentation and Source links because bugs can be written as well as our consumers and our contributors.
We could rephrase it to `Issues/Feature requests/Suggestions` which will invite our users to not only report bugs but also involve in the development of Precaution with ideas.

File: docs/index.md [hunk 2]
```diff
@@ -19,18 +19,31 @@ GitHub integration is made through the GitHub app interface and the checks API (
...
+* Go files via [Gosec](https://github.com/securego/gosec)
+* JavaScript and TypeScript via [TSLint](https://github.com/palantir/tslint) and [tslint-config-security](https://github.com/webschik/tslint-config-security)
+* Python files via [Bandit](https://github.com/PyCQA/bandit)
+
+New languages may be added in future, please file an [issue](https://github.com/vmware/precaution/issues) for your language/linter of choice.
 
 * Documentation: [vmware/precaution/docs](https://vmware.github.io/precaution/)
 * Source: [vmware/precaution](https://github.com/vmware/precaution)
-* Bugs: [vmware/precaution/issues](https://github.com/vmware/precaution/issues)
+<!-- * Bugs: [vmware/precaution/issues](https://github.com/vmware/precaution/issues) -->
```
Comment (joshuagl) [inline docs/index.md]: A general remark; don't use code comments to remove lines of code, just delete them. The revision control history (git) is able to show us what removed and, when paired with a decent commit message, why.

File: docs/index.md (hunk 1 above)
Comment (joshuagl) [reply docs/index.md]: Agreed. Filing issues/feature requests is a contribution, but we would like to encourage everyone to file them not just those who feel like they satisfy the label of contributor.

File: docs/index.md [hunk 3]
```diff
@@ -19,18 +19,31 @@ GitHub integration is made through the GitHub app interface and the checks API (
...
+
+## Try Precaution
 
-## Additional documentation
+Check out the several sections and you can continue with your project but with it being secure.
```
Comment (joshuagl) [inline docs/index.md]: This sentence is a little confusing, perhaps something like:
```suggestion
Check out the sections below to use Precaution to help secure your project.
```

**Comment by MVrachev** [comment]
I noticed another thing: @Stanislava101  you created a new file `docs/redirect_github_webhooks` but where do you reference it?
Why do you need it?
I think this file belongs to the `Setting up for manual deployment` doc.

**Comment by Stanislava101** [comment]
> I noticed another thing: @Stanislava101 you created a new file `docs/redirect_github_webhooks` but where do you reference it?
> Why do you need it?
> I think this file belongs to the `Setting up for manual deployment` doc.

Before it was there but I thought that if it is more separated it is more clear. My idea is in `Setting up a manual deployment` file to have the steps needed for Setting up a manual deployment and redirect Github webhooks to be in another file.

**Comment by MVrachev** [comment]
> Before it was there but I thought that if it is more separated it is more clear. My idea is in `Setting up a manual deployment` file to have the steps needed for Setting up a manual deployment and redirect Github webhooks to be in another file.

Yes, but Redirecting the GitHub webhooks to your local machine is essential for your local development. That's why it's better if it stays where it is.
Currently, in the documentation, it doesn't seem like this is so important.
I will fix that.

**Comment by MVrachev** [comment]
> Yes, but Redirecting the GitHub webhooks to your local machine is essential for your local development. That's why it's better if it stays where it is.
> Currently, in the documentation, it doesn't seem like this is so important.
> I will fix that.

Added pr https://github.com/vmware/precaution/pull/252 which fixes that.
